src/main/java/com/google/sps/scheduler/Scheduler.java

```diff
@@ -95,24 +94,12 @@
     while (dayEndEpochMilliseconds - lastEnd >= DEFAULT_DURATION_IN_MILLISECONDS &&
         tasksIterator.hasNext()) {
       DateTime startTime = epochToDateTime(lastEnd, timeZone);
+      Task task = tasksIterator.next();
+      task.setDue(startTime.toStringRfc3339());
+      scheduledTasks.add(task);
       lastEnd += DEFAULT_DURATION_IN_MILLISECONDS;
```

Comment:
just quickly glancing at this, it looks like almost a duplicate of the while loop above. Maybe there is some room for consolidation here.

Reply:
Yes, when I wrote that I thought of creating another method and I didn't do that because there are multiple objects that are updated, but now that I've looked at it better, the only thing that needs to be returned is `lastEnd`, so I can do it.